### PR TITLE
Create VCDM Explainer

### PR DESCRIPTION
**Pull request by TzviyaSiegman**
A first attempt at an Explainer for VCDM. Many gaps. Your feedback welcome.

### CODE REVIEW

**Comment by msporny** [comment]
Awesome @TzviyaSiegman -- thank you!

File: VCDMExplainer.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#Verfiable Credentials Data-Model Explainer
```
Comment (davidlehn) [inline VCDMExplainer.md]: Need a space after the heading `#` marks throughout the file.

Comment (TzviyaSiegman) [reply VCDMExplainer.md]: Thanks. I'll get these when I add more information.

Comment (davidlehn) [reply VCDMExplainer.md]: Got the rest, missed this one. ;-)

**Comment by msporny** [comment]
This is a good enough first cut to get merged in. Please provide additional PRs to improve/complete the document.

**Pull request merged**